pkg/netset/ipblock.go

```diff
@@ -396,6 +464,18 @@ func GetCidrAll() *IPBlock {
 	return res
 }
 
+// GetFirstIPAdress returns IPBlock object of 0.0.0.0
+func GetFirstIPAdress() *IPBlock {
```

Comment:
```suggestion
func GetFirstIPAddress() *IPBlock {
```
No spell checker in your IDE?